addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -239,7 +239,7 @@
     <!-- Selection Field -->
     <t t-name="website.form_field_selection">
         <t t-call="website.form_field">
-            <t t-if="!field.records">
+            <t t-if="!field.records || field.records.length == 0">
```

Comment:
I believe JS people prefer `!field.records.length` (instead of `== 0`) - but not sure in `t-if`. And if kept, maybe `===` ?